src/components/snackbar/index.js

```diff
@@ -29,7 +29,7 @@ const SnackbarProgrammatic = {
             delete params.message
         }
         const propsData = merge(defaultParam, params)
-        propsData.replace = true
+        propsData.queue = true
         const vm = typeof window !== 'undefined' && window.Vue ? window.Vue : localVueInstance || VueInstance
```

Comment:
Are you forcing it to be true??

Reply:
It was a mistake I made when  I tried to set `queue: true` as default in snackbars. I fixed it moving the `queue` property to the `defaultParams` object.